plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -93,179 +102,328 @@ function executeCLI($scriptName, &$args) {
+				if (!$genre) {
+					echo __('plugins.importexport.csv.noGenre') . "\n";
+					exit();
+				}
 
-					// we need a Genre for the files.  Assume a key of MANUSCRIPT as a default.
-					$genre = $genreDao->getByKey('MANUSCRIPT', $press->getId());
+				$authorGroup = $userGroupDao->getDefaultByRoleId($press->getId(), ROLE_ID_AUTHOR);
 
-					if (!$genre) {
-						echo __('plugins.importexport.csv.noGenre') . "\n";
-						exit();
-					}
-					if (!$authorGroup) {
-						echo __('plugins.importexport.csv.noAuthorGroup', array('press' => $pressPath)) . "\n";
-						continue;
-					}
-					if (in_array($locale, $supportedLocales)) {
-						$submission = $submissionDao->newDataObject();
-						$submission->setContextId($press->getId());
-						$submission->setUserId($user->getId());
-						$submission->stampLastActivity();
-						$submission->setStatus(STATUS_PUBLISHED);
-						$submission->setWorkType($isEditedVolume == 1?WORK_TYPE_EDITED_VOLUME:WORK_TYPE_AUTHORED_WORK);
-						$submission->setCopyrightNotice($press->getLocalizedSetting('copyrightNotice'), $locale);
-						$submission->setLocale($locale);
-
-						$series = $seriesDao->getByPath($seriesPath, $press->getId());
-						if ($series) {
-							$submission->setSeriesId($series->getId());
+				if (!$authorGroup) {
+					echo __('plugins.importexport.csv.noAuthorGroup', array('press' => $pressPath)) . "\n";
+					continue;
+				}
+
+				$submission = $submissionDao->newDataObject();
+				$submission->setContextId($press->getId());
+				$submission->stampLastActivity();
+				$submission->stampModified();
+				$submission->setStatus(STATUS_PUBLISHED);
+				$submission->setWorkType($isEditedVolume == 1 ? WORK_TYPE_EDITED_VOLUME : WORK_TYPE_AUTHORED_WORK);
+				$submission->setCopyrightNotice($press->getLocalizedSetting('copyrightNotice'), $locale);
+				$submission->setLocale($locale);
+				$submission->setStageId(WORKFLOW_STAGE_ID_PRODUCTION);
+				$submission->setSubmissionProgress('');
+				$submission->setTitle($title, $locale);
+				$submission->setData('abstract', $abstract, $locale);
+
+				// $submission->setUserId($user->getId()); // REVIEW
+
+				$series = $seriesPath ? $seriesDao->getByPath($seriesPath, $press->getId()) : null;
+				if ($series) {
+					$submission->setSeriesId($series->getId());
+				}
+
+				$submissionId = $submissionDao->insertObject($submission);
+
+				$publication = $publicationDao->newDataObject();
+				$publication->setData('submissionId', $submissionId);
+				$publication->setData('version', 1);
+				$publication->setData('status', STATUS_PUBLISHED);
+				$publication->setData('datePublished', Core::getCurrentDate());
+				$publicationId = $publicationDao->insertObject($publication);
+
+				$submission->setData('currentPublicationId', $publicationId);
+				$submissionDao->updateObject($submission);
+
+				$contactEmail = $press->getContactEmail();
+				$authorString = trim($authorString, '"'); // remove double quotes if present.
```

Comment:
This should be `authorsString` to distinguish from `authorString`, which is also used below for a single author.